website/docs/d/user.md

```diff
@@ -28,63 +28,68 @@ The following arguments are supported:
 
 In addition to the above arguments, the following attributes are exported:
 
-* `org_unit_path`
+* `org_unit_path` - OrgUnit of User.
 
-* `aliases`
+* `aliases` - List of aliases.
 
-* `agreed_to_terms`
+* `agreed_to_terms` - Indicates if user has agreed to terms.
 
-* `change_password_next_login`
+* `change_password_next_login` - Boolean indicating if the user should
+  change password in next login
 
-* `creation_time`
+* `creation_time` - User's G Suite account creation time.
 
-* `customer_id`
+* `customer_id` - CustomerId of User.
 
-* `deletion_time`
+* `deletion_time` - User's G Suite account deletion time.
 
-* `etag`
+* `etag` - ETag of the resource.
 
-* `include_in_global_list`
+* `include_in_global_list` - Boolean indicating if user is included in
+  Global Address List.
 
-* `is_ip_whitelisted`
+* `is_ip_whitelisted` - Boolean indicating if ip is whitelisted.
 
-* `is_admin`
+* `is_admin` - Boolean indicating if the user is admin.
 
-* `is_delegated_admin`
+* `is_delegated_admin` - Boolean indicating if the user is delegated admin.
 
-* `2s_enforced`
+* `2s_enforced` - Is 2-step verification enforced.
 
-* `2s_enrolled`
+* `2s_enrolled` - Is enrolled in 2-step verification.
 
-* `is_mailbox_setup`
+* `is_mailbox_setup` - Is mailbox setup.
 
-* `last_login_time`
+* `last_login_time` - User's last login time.
 
-* `name` - contains a set of `family_name`, `full_name`, `given_name`.
+* `name` - User's name.
+  contains a set of `family_name`, `full_name`, `given_name`.
 
-* `password`
+* `password` - User's password.
 
-* `hash_function` - md5, sha-1 and crypt.
+* `hash_function` - Hash function name for password. Supported are MD5,
+  SHA-1 and crypt
 
 * `posix_accounts` - contains a list of sets containing `account_id`, `gecos`,
   `gid`, `home_directory`, `shell`, `system_id`, `primary`, `uid`, `username`.
 
-* `ssh_public_keys` - contains a list of sets containing `expiration_time_usec`,
-  `key`, `fingerprint`.
+* `ssh_public_keys` - SSH public keys of the user.
+  contains a list of sets containing `expiration_time_usec`, `key`, `fingerprint`.
 
-* `is_suspended`
+* `is_suspended` - Indicates if user is suspended.
 
-* `suspension_reason`
+* `suspension_reason` -  Suspension reason if user is suspended.
 
-* `recovery_email`
+* `recovery_email` - Recovery email of the user.
 
-* `recovery_phone`
+* `recovery_phone` - Recovery phone of the user.
 
-* `custom_schema` - contains a schema of `name` and `value`.
+* `custom_schema` - Custom fields of the user.
 
-* `external_ids` - contains a list of sets containing `custom_type`, `type` and
-  `value`.
+* `external_ids` - A list of external IDs for the user, such as an employee or network ID. 
+  contains a list of sets containing `custom_type`, `type` and `value`.
 
-* `organizations` - contains a list of sets containing `cost_center`,
+* `organizations` - A list of organizations the user belongs to.
+  contains a list of sets containing `cost_center`,
```

Comment:
The above was copied from the URL:
https://github.com/googleapis/google-api-go-client/blob/master/admin/directory/v1/admin-gen.go#L3665